spec/integration/apply_compile_spec.rb

```diff
@@ -111,6 +111,24 @@ def get_notifies(result, future = false)
       expect(warn.count).to eq(1)
     end
 
+    it 'evaluates a node definition matching the node name' do
+      pending "puppet does not allow node definitions in plans"
+      result = run_cli_json(%w[plan run basic::node_definition] + config_flags)
```

Comment:
I think we need to add the `basic/plans/node_definition.pp` fixture too for this to pass once Puppet changes are available.

Reply:
Hah thanks I guess I forgot to commit those files but the test is pending so it was fine anyway 🙃